test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Inherits.cshtml

```diff
@@ -0,0 +1,2 @@
+﻿@using RazorCodeGen
+@inherits MyBasePage<string> base
```

Comment:
Generally is a View you'd inherit from `MyBasePage<TModel>` because we expect there to be a `ViewDataDictionary<TModel>` property defined on the class.

I think we need a case where you inherit from `MyBasePageForViews<TModel>` with a traditional MVC view and a case where you inherit from `MyBasePageForPages` for an `@page`.